service/k8sclient/kubectl/kubectl.go

```diff
@@ -38,8 +38,9 @@ import (
 )
 
 const (
-	// FIXME: change to a real default kubectl.
-	defaultKubectl = "minikube kubectl --"
+	kubeconfigFileName      = "kubeconfig.json"
+	defaultPmmServerKubectl = "/opt/dbaas-tools/bin/kubectl-1.16" // not in $PATH yet
```

Comment:
It will not be in PATH. If say otherwise somewhere, please update that place.

Reply:
Looks like I incorrectly understood your words. I'll remove this comment.

Reply:
No, it was supposed to be in $PATH, but that was changed